lms/templates/instructor/instructor_dashboard_2/metrics.html

```diff
@@ -146,6 +146,30 @@
         metrics_overlay.find('.metrics-student-grades').show();
         metrics_overlay.show();
       });
+      
+      // Click handler for x axis major ticks
+      $('.metrics-container').on("click", '.stacked-bar-graph-axis .tick.major', function () {
```

Comment:
We should move this to an external JS script file.

I know there's already javascript in this HTML template, but I'd really like to see us stop digging any deeper here.

Adding inline JS like this contributes to our terrible site speed issues;
this never gets minified and is sent across the wire on each load.